components/admin_console/feature_discovery/features/images/announcement_banner_svg.tsx

```diff
@@ -5,351 +5,545 @@ import React from 'react';
 
 const AnnouncementBannerSVG = () => (
     <svg
-        width='272'
+        width='172'
         height='220'
-        viewBox='0 0 272 220'
+        viewBox='0 0 172 220'
         fill='none'
         xmlns='http://www.w3.org/2000/svg'
     >
         <path
-            opacity='0.08'
-            fillRule='evenodd'
-            clipRule='evenodd'
-            d='M271.607 9.93672C271.611 9.17488 271.466 8.41969 271.181 7.71431C270.895 7.00893 270.474 6.36719 269.942 5.82577C269.41 5.28434 268.778 4.85385 268.081 4.55887C267.384 4.2639 266.636 4.11023 265.88 4.10666H10.3351C8.8085 4.11491 7.34763 4.73366 6.2736 5.8269C5.19957 6.92014 4.60028 8.3984 4.60749 9.93672V164.267H4.60742V182.548C4.60742 184.079 5.21087 185.547 6.285 186.63C7.35914 187.712 8.81598 188.32 10.335 188.32H106.02L103.134 213.321H172.254L168.459 188.32H263.496C268.885 188.324 271.611 186.112 271.607 180.31C271.607 179.552 271.607 164.267 271.607 164.267M271.607 9.93672L271.607 164.267L271.607 9.93672Z'
-            fill='var(--center-channel-color)'
+            d='M171.521 59.137C171.521 53.3935 165.803 47.6499 160.086 47.6499H11.4347C5.71737 47.6499 0 53.3935 0 59.137V168.33H171.521V59.137Z'
+            fill='#2D3039'
         />
         <path
-            opacity='0.07'
-            d='M134 218.24C202.544 218.24 258.111 216.795 258.111 215.013C258.111 213.231 202.544 211.787 134 211.787C65.455 211.787 9.88867 213.231 9.88867 215.013C9.88867 216.795 65.455 218.24 134 218.24Z'
-            fill='var(--center-channel-color)'
+            d='M0 168.33V179.817C0 185.561 5.71737 191.304 11.4347 191.304H160.086C165.803 191.304 171.521 185.561 171.521 179.817V168.33H0Z'
+            fill='#DDDFE4'
         />
         <path
-            opacity='0.1'
-            d='M169.078 213.727H134.665L100.184 213.283L100.389 211.504H168.737L169.078 213.727Z'
-            fill='var(--center-channel-color)'
+            d='M62.8914 197.026C62.8914 208.513 51.4567 208.513 40.0654 208.513H131.543C120.108 208.513 108.717 208.513 108.717 197.026V191.26H62.9784L62.8914 197.026Z'
+            fill='#C6C9D2'
```

Comment:
I think this and others should be css variables. @pablovelezvidal right?

Reply:
I had the same question and discussed with UX [here](https://community-daily.mattermost.com/private-core/pl/o7ht3xaf3pbs9rgoqe4h99whhh). The decision was that there's not a desire to override different themes for these pictures in particular, so no desire to turn into CSS variables.

Reply:
@neallred acknowledged